airflow/migrations/versions/c306b5b5ae4a_switch_xcom_table_to_use_run_id.py

```diff
@@ -132,6 +134,10 @@ def downgrade():
     op.create_table("__airflow_tmp_xcom", *_get_old_xcom_columns())
 
     xcom = Table("xcom", metadata, *_get_new_xcom_columns())
+
+    # Remoe XCom entries from mapped tis.
+    delete(xcom).where(xcom.c.map_index != -1)
```

Comment:
This doesn't go via alembic's op which means it doesn't get captured in to `--sql` mode.

Reply:
```suggestion
    op.execute(xcom.delete().where(xcom.c.map_index != -1))
```

Reply:
`airflow db downgrade -s --revision a3bcd0914482` to get the SQL.

Reply:
Actually the way I wrote it the query won’t get executed at all 🤪

Reply:
:lolsob: